blocks/identity-block/components/login/index.test.jsx

```diff
@@ -12,7 +12,7 @@ const defaultParams = {
 	loggedInPageLocation: "/account-2/",
 };
 
-const Test = (props) => {
+function Test(props) {
```

Comment:
Why did this have to change here, but was left the same in the other PR?

Reply:
I had the same question but for some reason on this PR it gave me some linting errors in the test file. Not sure what changed from the 2.1.2 and 2.3.0 but it gave me this linting error and I did a lint:fix and this is what it generated.